backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -450,10 +450,9 @@
   remove the leading /akvo/lumen and return paths as demo/admin."
   [{:keys [api-root user-id-cache jwt-paths-cache paths-cache-seconds connection-manager monitoring] :as keycloak} {:keys [email iat]}]
   (prometheus/with-duration (registry/get (:collector monitoring) :app/auth-allowed-paths {})
-    (when-let [headers (request-headers keycloak)]
```

Comment:
We are not checking now that the result of `request-headers` is nil. Is this on purpose?

Reply:
now it's better, tx!